### PR TITLE
mat-card for a car

### PR DESCRIPTION
**Pull request by an-say**
Using angular material cards for a car. Design saved

### CODE REVIEW

File: src/app/cars/car/car.component.scss [hunk 1]
```diff
@@ -1,70 +1,72 @@
...
+.car-title {
     float: left;
     padding-left: 5px;
     border-left: 3px solid green;
 }
 
-.content .rating {
-    float: right;
+.car-rate {
+    float: right !important;
```
Comment (sverigo) [inline src/app/cars/car/car.component.scss]: please don't use !important (probably can be fixed using selectors weight)

File: src/app/cars/car/car.component.scss [hunk 2]
```diff
@@ -1,70 +1,72 @@
-.image {
-    width: 50%;
+.car-card{
```
Comment (sverigo) [inline src/app/cars/car/car.component.scss:3]: use scss features for styles files

File: src/app/cars/car/car.component.scss [hunk 3]
```diff
@@ -1,70 +1,72 @@
...
     display: flex;
-    justify-content: center;
-    align-items: center;
+    padding: 0;
     overflow: hidden;
-    border-radius: 5px 0 0 5px;
+    min-width: 450px;
```
Comment (sverigo) [inline src/app/cars/car/car.component.scss]: This part of code breaks flexible design (need to support 800px width)

Comment (sverigo) [reply src/app/cars/car/car.component.scss]: [https://ibb.co/ZdSHkpY](https://ibb.co/ZdSHkpY)

File: src/app/cars/car/car.component.scss (hunk 2 above)
Comment (sverigo) [reply src/app/cars/car/car.component.scss:3]: disable outline for mat-card

File: src/app/cars/car/car.component.scss (hunk 3 above)
Comment (an-say) [reply src/app/cars/car/car.component.scss]: need to re-check

**Review by sverigo — APPROVED** [review]

**Pull request merged**